contributors/devel/development.md

```diff
@@ -279,20 +279,42 @@ To build binaries for all platforms:
 make cross
 ```
 
+#### Install etcd
+```
+hack/install-etcd.sh
+```
+
+#### Verification
+```
+# Run all hack/verify-*.sh scripts
+make verify
+```
+
 #### Test
 
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update-*.sh scripts
+# For small change, run "make verify" and then relevant scripts, for example:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+make update
+
 # Run every unit test
 make test
 
 # Run package tests verbosely
 make test WHAT=pkg/util/cache GOFLAGS=-v
+# Or use bazel (quote "..." when using zsh)
+bazel test //pkg/util/cache/...
 
 # Run integration tests, requires etcd
 make test-integration
 
+# (Optional) Build e2e test binaries
+make all WHAT=test/e2e/e2e.test
```

Comment:
This builds the `test/e2e/e2e.test` binary, which is a hack I used to run e2e test against a local cluster (hack/local-up-cluster.sh). However, `make test-e2e` builds a quick release which contains many other binaries, including this one. You can try running `go run hack/e2e.go -- -v --build` (the first step in `make test-e2e`) to see which binaries are built. 

Just remove this or move it to a section for e2e test for a local cluster (if we have one).